ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-stable-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -59,17 +59,8 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: aws-ipi-ovn-ipsec-amd-mixarch-day2-f14
-  cron: 42 5 14,28 * *
-  steps:
-    cluster_profile: aws-qe
-    env:
-      BASE_DOMAIN: qe.devcluster.openshift.com
-    test:
-    - chain: openshift-upgrade-qe-sanity-heterogeneous
-    workflow: cucushift-installer-rehearse-aws-ipi-ovn-ipsec
-- as: aws-ipi-ovn-ipsec-amd-mixarch-f14
-  cron: 10 8 6,20 * *
+- as: aws-ipi-ovn-ipsec-amd-mixarch-f28
```

Comment:
If we are align with openshift-openshift-tests-private-release-4.16__multi-stable.yaml, the job should be `aws-ipi-ovn-ipsec-arm-mixarch-f28`, right?